testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/grpc/GrpcToJaxrsTest.java

```diff
@@ -0,0 +1,805 @@
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.exporter.ZipExporter;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.google.protobuf.Any;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.core.Response;
+import jaxrs.example.CC1ServiceGrpc;
+import jaxrs.example.CC1_proto;
+import jaxrs.example.CC1ServiceGrpc.CC1ServiceBlockingStub;
+import jaxrs.example.CC1_proto.FormMap;
+import jaxrs.example.CC1_proto.FormValues;
+import jaxrs.example.CC1_proto.GeneralEntityMessage;
+import jaxrs.example.CC1_proto.GeneralReturnMessage;
+import jaxrs.example.CC1_proto.ServletInfo;
+import jaxrs.example.CC1_proto.gCookie;
+import jaxrs.example.CC1_proto.gHeader;
+import jaxrs.example.CC1_proto.gString;
+import jaxrs.example.CC1_proto.org_jboss_resteasy_example___CC2;
+import jaxrs.example.CC1_proto.org_jboss_resteasy_example___CC3;
+import jaxrs.example.CC1_proto.org_jboss_resteasy_example___CC4;
+import jaxrs.example.CC1_proto.org_jboss_resteasy_example___CC5;
+
+/**
+ * @tpSubChapter Jaxrs implementation
+ * @tpChapter Integration tests
+ * @tpTestCaseDetails RESTEASY-1531
+ * @tpSince RESTEasy 3.1.0
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+public class GrpcToJaxrsTest
+{
+   protected static final Logger log = Logger.getLogger(GrpcToJaxrsTest.class.getName());
+
+   @Deployment
+   public static Archive<?> deploy()
+   {
+      WebArchive war = TestUtil.prepareArchive(GrpcToJaxrsTest.class.getSimpleName());
+      war.addClass(io.grpc.netty.shaded.io.netty.channel.group.ChannelMatchers.class);
+      war.addClass(com.google.common.util.concurrent.internal.InternalFutureFailureAccess.class);
+      war.setManifest(new StringAsset("Manifest-Version: 1.0\n"
+//            + "Dependencies: com.google.guava services,org.jboss.resteasy.resteasy-grpc-provider services\n"));
+      + "Dependencies: com.google.guava services\n"));
+      war.merge(ShrinkWrap.createFromZipFile( WebArchive.class, TestUtil.resolveDependency("jaxrs.example:jaxrs.example.grpc:war:0.0.2-SNAPSHOT")));
+      TestUtil.addOtherLibrary(war, "org.jboss.resteasy:grpc-bridge:jar:6.1.0-SNAPSHOT");
+      TestUtil.addOtherLibrary(war, "com.github.javaparser:javaparser-symbol-solver-core:jar:3.24.2");
+      TestUtil.addOtherLibrary(war, "com.github.javaparser:javaparser-core:jar:3.24.2");
+      TestUtil.addOtherLibrary(war, "org.wildfly:wildfly-weld-jakarta:jar:26.1.0.Final");
+      TestUtil.addOtherLibrary(war, "org.wildfly:wildfly-weld-common-jakarta:jar:26.1.0.Final");
+      TestUtil.addOtherLibrary(war, "org.wildfly.core:wildfly-server:jar:18.1.0.Final");
```

Comment:
Why are these needed? Deployments should NEVER use these and they should definitely not be included in a deployment.

Reply:
Gone